tox.ini

```diff
@@ -15,7 +16,8 @@ envlist =
 [testenv]
 deps = {[base]testdeps}
 extras = plotly
-commands = pytest {posargs}
+commands =
+    pytest {posargs}
 
 [testenv:lint]
 basepython=python3.7
```

Comment:
Unrelated to this PR, but specifying `basepython=python3.7` in `lint` and `format` seems outdated, since the CI is running `py38-lint`.

Reply:
Rebased wrt the latest changes. It executes the correct testenvs/python combos now.